elements/pfe-cta/src/pfe-cta.js

```diff
@@ -13,42 +13,94 @@ class PfeCta extends PFElement {
     return "pfe-cta.html";
   }
 
```

Comment:
Did we intentionally leave `get schemaUrl() { return "pfe-cta.json"; }` out? If so, we should remove all of the `observedAttributes` and `attributeChangedCallback` code.

Reply:
Thanks Kyle!  I added it with the latest commit.